content/blog/github-token-scanning-service/index.md

```diff
@@ -10,8 +10,6 @@ meta_image: "feature.png"
 
 We are very excited to announce that we have partnered with GitHub to offer our users better protection for their Pulumi Access Tokens.
 
-## Pulumi Access Tokens
-
 By default, Pulumi users manage the state of their cloud infrastructure deployments using https://app.pulumi.com. This service provides state storage, concurrency control, audit history and access controls for both individuals and teams working with Pulumi.  Each user and service account can generate one or more Pulumi Access Tokens to be used to authenticate with this service.  These access tokens can be used on both local development machines, as well as in CI/CD systems for automated infrastructure deployments.  These access tokens are sensitive secrets which should never be shared publicly, and in particular should never be committed to source control.
```

Comment:
Another thing Eric noticed:

Auto-links don't show up in summaries:

<img width="772" alt="Screen Shot 2019-08-19 at 2 12 34 PM" src="https://user-images.githubusercontent.com/710598/63299803-7fb01680-c28b-11e9-83ab-3a368e4dcd9a.png">

But explicit links in summaries of other posts do appear to show up:

<img width="784" alt="Screen Shot 2019-08-19 at 2 13 50 PM" src="https://user-images.githubusercontent.com/710598/63299877-b0904b80-c28b-11e9-8aa1-756fa812b9c1.png">

I think we may need to just explicitly make this a link so that it's both a link in the post itself and the summary:

```suggestion
By default, Pulumi users manage the state of their cloud infrastructure deployments using [https://app.pulumi.com](https://app.pulumi.com). This service provides state storage, concurrency control, audit history and access controls for both individuals and teams working with Pulumi.  Each user and service account can generate one or more Pulumi Access Tokens to be used to authenticate with this service.  These access tokens can be used on both local development machines, as well as in CI/CD systems for automated infrastructure deployments.  These access tokens are sensitive secrets which should never be shared publicly, and in particular should never be committed to source control.
```